### PR TITLE
removed undefined class from textfield label

### PR DESCRIPTION
**Pull request by codingcircus**
Just removes an undefined class inside TextField Element Labels

### CODE REVIEW

**Comment by codingcircus** [comment]
I also fixed the missing multiple choice button styles now

File: components/MultipleChoice/index.jsx [hunk 1]
```diff
@@ -85,7 +85,14 @@ export default class MultipleChoice extends Component {
...
     const {inputStyle, requiredLabel} = this.props;
-    const classNames = [formGroup, formGroupMultipleChoice, sharedStyles[inputStyle]];
+
+    const inputStyles = inputStyle.split(' ');
+
+    const classNames = [formGroup, formGroupMultipleChoice];
+
+    // Inline Styles is shared in global index.scss, buttons is a local style
+    if (inputStyles.includes('buttons')) classNames.push(styles.buttons);
+    if (inputStyles.includes('inline')) classNames.push(sharedStyles.inline);
```
Comment (kaytcat) [inline components/MultipleChoice/index.jsx]: We could move this into a function to reduce duplicate code

File: components/TextField/index.jsx [hunk 2]
```diff
@@ -220,11 +219,13 @@ export default class TextField extends Component {
     // clickable element like a link or button inside a label
     // However to also add the labelContainer class, we need to return a cloned
     // element and not just the label - element itself
+    const classNames = [sharedStyles.labelContainer];
```
Comment (kaytcat) [inline components/TextField/index.jsx]: Need space after

File: components/TextField/index.jsx [hunk 3]
```diff
@@ -220,11 +219,13 @@ export default class TextField extends Component {
...
+    if (label.props.className) classNames.push(label.props.className);
```
Comment (kaytcat) [inline components/TextField/index.jsx]: Need space here too

**Pull request merged**